ext/scripts/security_scan/run.sh

```diff
@@ -39,5 +40,6 @@ function _run_security_scanners() {
 }
 
 security_scanners=$SECURITY_SCANNERS
+#shellcheck disable=SC2086
```

Comment:
Must be ignored, we need to handle $security_scanners as array and want to split on space

Reply:
yep, maybe add a comment to ignore in the code

Reply:
Done